docs/tutorials/developer-liftoff-rust/level-1/1.2-rust-dev-env.mdx

```diff
@@ -0,0 +1,139 @@
+:::
+
+## Setting up a local developer environment
+
+### Confirm you have a connection to the internet
+
+To follow along with the Developer Liftoff and develop on ICP, you will need a connection to the internet.
+
+#### Why does this matter?
+
+You will need an internet connection to download a few different tools and packages. You will also need an internet connection whenever you plan to deploy your project to the mainnet. You do not need an internet connection to deploy a project to your local development environment.
+
+### Confirm you have access to a command line interface (CLI)
+
+Open a command line interface (CLI) window. This may be referred to as 'Terminal' or 'Shell' depending on your computer's operating system. In this documentation, this is often referred to as the 'terminal window.'
+
+#### Why does this matter?
+
+Certain tutorials in this series require a local developer environment to use tools such as `dfx` and `git`.
+
+### Download and install Node.js
+
+Download and install [Node.js](https://nodejs.org/en).
+
+#### Why does this matter?
+
+Node.js is used by the IC SDK to generate frontend code and dependencies.
+
+### Download and install the Rust toolchain
+
+Download and install the [Rust toolchain](https://www.rust-lang.org/tools/install).
+
+#### Why does this matter?
+
+The Rust toolchain is required to install Rust dependencies and compile Rust code.
+
+### Add the `wasm32-unknown-unknown` target
+
+```
+rustup target add wasm32-unknown-unknown
+```
+
+#### Why does this matter?
+
+Canisters written in Rust are compiled to the Wasm target `wasm32-unknown-unknown`.
+
+### Download and install the IC SDK
+
+The [IC SDK](https://github.com/dfinity/sdk) contains the CLI tool `dfx`. `dfx` is natively supported on Linux or macOS 12.\* Monterey or later.
+
+To install the IC SDK, first install `dfxvm`, a CLI tool used for installing and switching between different versions of `dfx`.
+
+Download and install the latest version of [`dfxvm`](https://github.com/dfinity/dfxvm) with the following command:
+
+```bash
+sh -ci "$(curl -fsSL https://internetcomputer.org/install.sh)"
+```
+
+If you are using a machine running Apple silicon, you will need to have [Rosetta](https://support.apple.com/en-us/HT211861) installed. You can install Rosetta by running `softwareupdate --install-rosetta` in your terminal.
+
+#### Why does this matter?
+
+The IC SDK is composed of several components that are required for developing on the Internet Computer. These components are:
+
+- `dfx`: The CLI tool used to interact with and develop canisters on ICP. Motoko is included in the installation of dfx.
+- `moc`: The Motoko runtime compiler.
+- `replica`: The software run by each node on the network. By having an instance of this software locally, you can deploy projects to a local environment.
```

Comment:
As of the latest release of dfx, the replica has been fully replaced with PocketIC for local dev and dfx no longer bundles the replica. The factoid most relevant to this article is that it simulates a multi-subnet IC network locally (which the replica did not do).